plutus-core/plutus-ir/test/transform/inline/letFunConstBool

```diff
@@ -1,3 +1,15 @@
+{-
```

Comment:
I love these so much!

Reply:
Many of the golden files are missing

Reply:
Sorry I was still working on them. It's a draft. I'll ping you when ready.